src/ng/sniffer.js

```diff
@@ -61,7 +65,7 @@ function $SnifferProvider() {
       // so let's not use the history API also
       // We are purposefully using `!(android < 4)` to cover the case when `android` is undefined
       // jshint -W018
-      history: !!($window.history && $window.history.pushState && !(android < 4) && !boxee),
+      history: !!(hasHistoryPushState && !(android < 4) && !boxee),
```

Comment:
> Is Chrome more special that android < 4 or boxee

Yeah, Chrome is actually very close to my heart :stuck_out_tongue: 

I put it this way for readability (avoiding `!!(!` and hitting the 100 chars per line limit, thus having to wrap). 
I'm fine having it either way.